create_infotexts.py

```diff
@@ -100,19 +101,42 @@ def load_places_mapping():
 
     return places_dict
 
+
+def create_commons_filename(metadata, fotonr):
+    """Creates commons filename according to https://phabricator.wikimedia.org/T156612 and ouputs to new_dict
```

Comment:
For multi-line docstrings the initial line should be on the line following the `"""`. It should also be more general wiht any additional details added to subsequent lines. (also see comments above) I.e. Here
```
"""
Create commons filename without file extension.

Construction is done according to https://phabricator.wikimedia.org/T156612.
:param ...
````